.github/workflows/label-checker.yml

```diff
@@ -0,0 +1,21 @@
+---
+name: Label Checker
+on:
+  pull_request:
+    types:
+      - opened
+      - synchronize
+      - reopened
+      - labeled
+      - unlabeled
+
+jobs:
+
+  check_labels:
+    name: Check labels
+    runs-on: ubuntu-latest
+    steps:
+      - uses: docker://agilepathway/pull-request-label-checker:latest
+        with:
+          one_of: bug,documentation,enhancement,feature,security,:test,>upgrade
```

Comment:
My concern is the labels in the list may not enough. For example, the PRs created by `dependabot`, they will be labeled as `dependencies` automatically. Looks like if not having that label, those PRs will fail the check.

Reply:
Thanks for raising the point. I know this is not a complete list and would like inputs on what else can be included. It helps break the PRs atleast into major topics that we currently track. We can always add additional and follow a mapping mechanism in the solution we develop further - i.e. documentation, javadocs -> Documentation